### ISSUE TITLE
Can we filter by cards again?

### ISSUE THREAD

**Issue by jw9364** [cardona150/MlbTheShow18MarketAssistant#13]
I was easily making 200K stubs a day prior to this may update, now i cant get anything working.. I only want to swap player cards, they sell the fastest.

Is there anyway we can change the new screen to include cards between 78 and 80, for example, with the cost from 1k-5k? I tried to change the url to include the filter but the assistant wont handle it

If not possible, is there anyway i can download the pre-May 29th release? Again, i appreciate you doing this. I wish i had more time with the previous release, only a few days away from Vlad

**Comment by damagedphantom** [comment]
I'd love to have the last version back as well. The ability to filter what I'm seeing definitely helped the profits. Now I'm constantly seeing stuff that's basically just garbage and taking up screen space. Hopefully we can get that pre 5/29 download!

**Comment by jw9364** [comment]
I believe this version will be an improvement on the previous version once we get the filter options back

I know the dev has done a ton of work with this and i look forward to see what he has in store!

**Comment by damagedphantom** [comment]
Yes, this version is very nice. Literally the only issue I'm having is not being able to filter what I'm seeing.

I, too, appreciate everything the developer has done for us. 

Not to give him any ideas, but this is something I'd willingly spend $3-5 on, assuming he'd update with each years game.

**Comment by aasatt** [comment]
I agree with this also.

For example I would just like to be able to add my own custom query at the end: `type=Souvenir&team=Marlins`

I don't need the GUI to support custom filters but prior to v11 this worked and now it is broken.

Nevertheless, thank you for creating this for the community.
